app/src/main/res/values-it/strings.xml

```diff
@@ -122,100 +122,162 @@
     <string name="account_settings_active">Attivi</string>
-    <string name="account_settings_hot">Di tendenza</string>
+    <string name="account_settings_hot">Popolari</string>
     <string name="account_settings_new">Nuovi</string>
-    <string name="account_settings_topday">Top giorno</string>
-    <string name="account_settings_topweek">Top settimana</string>
-    <string name="account_settings_topmonth">Top mese</string>
-    <string name="account_settings_topyear">Top anno</string>
-    <string name="account_settings_topall">Top di tutti i tempi</string>
-    <string name="account_settings_mostcomments">Più commenti</string>
-    <string name="account_settings_newcomments">Nuovi commenti</string>
-    <string name="account_settings_default_sort_type">Ordinamento predefinito</string>
+    <string name="account_settings_topday">TopGiorno</string>
+    <string name="account_settings_topweek">TopSettimana</string>
+    <string name="account_settings_topmonth">TopMese</string>
+    <string name="account_settings_topyear">TopAnno</string>
+    <string name="account_settings_topall">TopSempre</string>
+    <string name="account_settings_mostcomments">PiùCommentati</string>
+    <string name="account_settings_newcomments">NuoviCommenti</string>
+    <string name="account_settings_default_sort_type">Ordinamento Predefinito</string>
     <string name="account_settings_show_nsfw">Mostra NSFW</string>
-    <string name="account_settings_show_avatars">Mostra avatar</string>
-    <string name="account_settings_show_read_posts">Mostra post letti</string>
-    <string name="account_settings_bot_account">Account bot</string>
-    <string name="account_settings_show_bot_accounts">Mostra account bot</string>
-    <string name="account_settings_show_scores">Mostra punteggi</string>
-    <string name="account_settings_show_notifications_for_new_posts">Mostra notifiche per nuovi post</string>
-    <string name="account_settings_send_notifications_to_email">Invia notifiche via email</string>
-    <string name="account_settings_save_settings">Salva impostazioni</string>
-    <string name="account_settings_activity_account_settings">Impostazioni account</string>
+    <string name="account_settings_show_avatars">Mostra Avatar</string>
+    <string name="account_settings_show_read_posts">Mostra Post Già Letti</string>
+    <string name="account_settings_bot_account">Impostati Come Bot</string>
+    <string name="account_settings_show_bot_accounts">Mostra Bot</string>
+    <string name="account_settings_show_scores">Mostra Punteggi</string>
+    <string name="account_settings_show_notifications_for_new_posts">Invia Notifiche per Nuovi Post</string>
+    <string name="account_settings_send_notifications_to_email">Invia Notifiche via Email</string>
+    <string name="account_settings_save_settings">Salva Impostazioni</string>
+    <string name="account_settings_activity_account_settings">Impostazioni Account</string>
```

Comment:
In a previous translation  you replace account with profilo, but here it's account again. Which one?

Reply:
Fixed using Profilo everywhere